assembl/views/api2/discussion.py

```diff
@@ -822,3 +827,78 @@ def test_results(request):
              permission=P_READ)
 def test_sentry(request):
     raise RuntimeError("Let's test sentry")
+
+
+@etalab_discussions.post(permission=P_SYSADMIN)
```

Comment:
You can always apply multiple view_config decorators. Cornice just creates dyanamic view_config decorators. No side effect.